package.json

```diff
@@ -63,7 +63,7 @@
     "husky": "^3.0.2",
     "lerna": "^3.22.1",
     "lint-staged": "^9.2.1",
-    "madge": "^4.0.0",
+    "madge": "^4.0.2",
```

Comment:
Is this really needed? madge seems to be dev tool to generate a graph of dependencies

Reply:
Same for all `package-lock.json` changes

Reply:
I didn't intentionally do it maybe npm upgraded it automatically when I compiled the code.

Reply:
Most likely, yes, but please check what you're commiting

Reply:
Is it advised for me to close this PR and create a new one in which only NoteList.tsx is changed?